src/shared_modules/content_manager/include/contentManager.hpp

```diff
@@ -27,10 +27,23 @@
 #define EXPORTED
 #endif
 
+/**
+ * @brief ContentModule class.
+ *
+ */
 class EXPORTED ContentModule final : public Singleton<ContentModule>
 {
 public:
+    /**
+     * @brief Start module facade.
+     *
+     */
     void start(const std::function<void(const modules_log_level_t, const std::string&)>& /*logFunction*/);
+
+    /**
+     * @brief Stop module facade.
+     *
+     */
```

Comment:
Question: Are these descriptions correct for this class? I see that there is another class named `ContentModuleFacade`

Reply:
It is meant to be correct because that `start` method does the following:
```C++
void ContentModule::start(const std::function<void(const modules_log_level_t, const std::string&)>& /*logFunction*/)
{
    ContentModuleFacade::instance().start();
}
```

Reply:
Same for the `stop` method:
```C++
void ContentModule::stop()
{
    ContentModuleFacade::instance().stop();
}
```